### PR TITLE
Feature/cl 393

### PR DESCRIPTION
**Pull request by talktocory**
Hi @JochenLinnemann ,

I've got the entity framework implemented and have incorporated the new leafletjs marker functionality.  I have yet to add a save method for marker data that gets added to the map.  Somehow it's going to have to work in the map secret for authorization purposes.  Not sure yet how that will work.

Cory

### CODE REVIEW

**Comment by JochenLinnemann** [comment]
Will you look at the conflicting file and let me know when the conflict is fixed? Thanks!

**Comment by talktocory** [comment]
Hi @JochenLinnemann 

The TileCreationService is structurally quite different from the original.  It is now a singleton background service that processes tile creation requests on a timer.  This was done to satisfy one of the requirements to ensure that the service creates all required tiles even if it restarts mid-processing.  

Let me know what you think.

Cory

File: src/CampaignKit.WorldMap/Controllers/MapController.cs [hunk 1]
```diff
@@ -24,289 +24,278 @@
...
+			else
+				return RedirectToAction(nameof(Show), new { map.MapId, map.Secret, ShowProgress = true });
+
+			return View();
+		}
+
+		public async Task<IActionResult> Index()
+		{
+			var model = await _mapDataService.FindAll();
+			model.OrderByDescending(m => m.CreationTimestamp);
```
Comment (JochenLinnemann) [inline src/CampaignKit.WorldMap/Controllers/MapController.cs]: Is OrderByDescending() working in place or does it return the sorted model?
```suggestion
 			model = model.OrderByDescending(m => m.CreationTimestamp);
```

**Review by JochenLinnemann — COMMENTED** [review]
I will have to test this :-)

**Comment by talktocory** [comment]
@JochenLinnemann - I think I know what's going on here.  I'm just getting used to the GitHub/GitFlow process and I inadvertently jumped the gun on the pull request.  I was thinking that I needed to do a pull request after each feature was completed but really I'm making a broad set of connected changes that aren't all complete yet.  I didn't want to lose the in-flight work and wanted to get it submitted.  What I should have done (I think) is completed the feature and committed it to the Development branch and then started a new feature to work on other changes.  Once those are all done and stabilized I'll submit the pull request.  

So....  Maybe it's best to reject this pull request and I'll start another feature on the development branch for map data persistence.  

Let me know what you think.

Cory

**Pull request closed without merging**